cmd/monaco/deploy/deploy.go

```diff
@@ -51,9 +52,12 @@ func deployConfigs(ctx context.Context, fs afero.Fs, manifestPath string, enviro
 		return err
 	}
 
-	ok := verifyEnvironmentGen(ctx, loadedManifest.Environments.SelectedEnvironments, dryRun)
-	if !ok {
-		return fmt.Errorf("unable to verify Dynatrace environment generation")
```

Comment:
FYI, I removed this line, as it had no additional value to the returned error now